src/hubbleds/free_response.py

```diff
@@ -11,27 +11,27 @@
 @dataclasses.dataclass
 class FreeResponse:
     tag: str = dataclasses.field(init=True)
-    _response: Reactive[str] = dataclasses.field(default_factory = lambda: Reactive(""))
-    _initialized: Reactive[bool] = dataclasses.field(default_factory = lambda: Reactive(True))
+    _response: str = dataclasses.field(default_factory = lambda: "")
```

Comment:
Is there a reason that these are factories and not just regular fields?

Reply:
I suppose not anymore. Previously (when testing `MCScore`), I found the each copy of the `MCScore` dataclass would have the same `Reactive` variable in it. But since these aren't reactive, I will change them back